app/assets/javascripts/admin/addon/routes/admin-emojis-settings.js

```diff
@@ -0,0 +1,20 @@
+import DiscourseRoute from "discourse/routes/discourse";
+import I18n from "discourse-i18n";
+import SiteSetting from "admin/models/site-setting";
+
+export default class AdminEmojisSettingsRoute extends DiscourseRoute {
+  queryParams = {
+    filter: { replace: true },
+  };
+
+  titleToken() {
```

Comment:
Need to add this to the overarching admin emojis route too, so in the title we get `Settings - Emojis - Admin`